backend/capellacollab/projects/toolmodels/diagrams/routes.py

```diff
@@ -128,14 +128,20 @@ async def get_diagram(
     file_path = f"diagram_cache/{parse.quote(diagram_uuid, safe='')}.svg"
 
     try:
-        file_or_artifact = await handler.get_file_or_artifact(
+        res_job_id, date, data = await handler.get_file_or_artifact(
```

Comment:
```suggestion
        _, _, data = await handler.get_file_or_artifact(
```

I don't know why ruff didn't catch it, but those two variables are unused.